src/modules/toolbox.rs

```diff
@@ -0,0 +1,69 @@
+use super::{Context, Module, RootModuleConfig};
+
+use crate::configs::toolbox::ToolboxConfig;
+use crate::formatter::StringFormatter;
+
+/// Creates a module with the current toolbox image
+///
+/// Will display the toolbox image if `$toolbox_NAME` is set.
+pub fn module<'a>(context: &'a Context) -> Option<Module<'a>> {
+    let toolbox_env = context.get_env("TOOLBOX_PATH")?;
+    let toolbox_name = context.get_env("NAME")?;
```

Comment:
Is set in our [Containerfile](https://github.com/containers/toolbox/blob/cfffb72fb0c275bfe32318c044a408603ca769f9/images/fedora/f36/Containerfile#L3) but we already have [reports](https://github.com/containers/toolbox/issues/188) of these variables interfering with other tools. While it is not possible to not use them for the foreseeable future due to the way Fedora OCI image machinery works, it is only specific to the Fedora images and they are not Toolbx-specific.

To read the name of the container, I'd read the file `/run/.containerenv` created by Podman (which is a dependency of Toolbx) and get the name from there. The content of the file can look like this:

```
engine="podman-3.4.2"
name="toolbox-dev"
id="236e7a414947759564bb2bc2d5cb3fc4a5999c7e8457bc763aaf5361a3f700c8"
image="registry.fedoraproject.org/fedora-toolbox:36"
imageid="3ef9a4eaf734b79fcafc0f846efb8a8e8850cfc81c66a9e8aea10c6637623a04"
rootless=1
graphRootMounted=1⏎
```

Reply:
noted, it's more elegant indeed.